src/test/regress/expected/add_coordinator.out

```diff
@@ -1,6 +1,10 @@
 --
 -- ADD_COORDINATOR
 --
+-- node trying to add itself without specifying groupid => 0 should error out
+SELECT master_add_node('localhost', :master_port);
+ERROR:  Node cannot add itself as a worker.
+HINT:  Add the node as a coordinator by using: SELECT citus_add_node(localhost, 57636, groupid => 0);
```

Comment:
for consistency with the other error message in AddNodeMetadata, can we suggest citus_set_coordinator_host here?

Reply:
But it seems we do not need `citus_set_coordinator_host` with `master_add_node`. because we add this message `if (!nodeAlreadyExists)`, whereas `citus_set_coordinator_host` happens `if (nodeAlreadyExists)`.

I think we can do that separately, but it looked confusing to see `citus_set_coordinator_host` when the node does not really exist.

I thought we could consider another PR which does a similar check but gives `citus_set_coordinator_host` notice/warning when ` `if (nodeAlreadyExists)`.

Do you think we should definitely have it here as well?

Reply:
We do not need citus_add_node if we use citus_set_coordinator_host. If the coordinator is not yet added to the metadata then citus_set_coordinator_host will add it.

I think we should at least be self-consistent in our error messages. Adding the node to itself also seems like a weird thing to do. Surely the coordinator knows it exists? Setting the coordinator host seems like a much more reasonable thing to do.

Reply:
> . If the coordinator is not yet added to the metadata then citus_set_coordinator_host will add it.

Ops, I didn't realize this. Given the name contains `set`, I was considering this as operating only if the node exists.

So, @gledis69 would you mind adding back `citus_set_coordinator_host` to the message, sorry for the extra iteration